confluence-slack-server-integration-plugin/src/main/java/com/atlassian/confluence/plugins/slack/spacetochannel/ao/AOEntityToChannelMapping.java

```diff
@@ -8,6 +8,8 @@
 @Preload("*")
 @Table(value = "EntitiesToChannels")
 public interface AOEntityToChannelMapping extends Entity {
+    String CHANNEL_ID_COLUMN = "CHANNEL_ID";
```

Comment:
Could you please elaborate what this lines does?

Reply:
I had to use this field to get AOEntityToChannelMapping in MutedChannelIdUpgradeTask001